src/codegen/generate-classes.ts

```diff
@@ -1860,10 +1854,7 @@ const JavaScriptCoreBindings = struct {
           output += `
         pub fn ${names.fn}(globalObject: *JSC.JSGlobalObject, callFrame: *JSC.CallFrame) callconv(JSC.conv) JSC.JSValue {
           if (comptime Environment.enable_logs) JSC.markBinding(@src());
-          return @call(.always_inline, ${typeName}.${fn}, .{globalObject, callFrame}) catch |err| switch (err) {
-            error.JSError => .zero,
-            error.OutOfMemory => globalObject.throwOutOfMemoryValue(),
-          };
+          return JSC.toJSHostFunction(${typeName}.${fn})(globalObject, callFrame);
```

Comment:
The inline was intentional. We cannot add extra functions to the call stack. Please revert this